### PR TITLE
cf581D- Solution added in Java and Python

### PR DESCRIPTION
**Pull request by shuklaji23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by shuklaji23** [comment]
Sure, @SansPapyrus683 here is the link to the accepted submissions on codeforces.

File: solutions/bronze/cf-581D.mdx [hunk 1]
```diff
@@ -89,4 +89,146 @@ int main() {
...
+			}
+			io.flush();
+			io.close();
+			return;
+		}
+
+		int n = Math.max(logos[0], Math.max(logos[2], logos[4]));
+		int h = -1;
+
+		char let[] = new char[] {'A', 'B', 'C'};
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-581D.mdx:131]:
```suggestion
		char[] let = new char[] {'A', 'B', 'C'};
```
same for other array declarations

File: solutions/bronze/cf-581D.mdx [hunk 2]
```diff
@@ -89,4 +89,146 @@ int main() {
...
+			char c = 'A';
+			io.println(n);
+			k = 0;
+			for (int j, i = 1; i < 6; i += 2, c++) {
+				for (j = 0; j < logos[i]; j++) {
+					for (k = 0; k < n; k++) { io.print(c); }
+					io.println();
+				}
+			}
+			io.flush();
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-581D.mdx:123]: flush not needed

File: solutions/bronze/cf-581D.mdx [hunk 3]
```diff
@@ -89,4 +89,146 @@ int main() {
...
+			for (k = 0; k < n; k++) { io.print(let[2]); }
+			io.println();
+		}
+		k = 0;
+		for (int j = h; j < n; j++) {
+			for (k = 0; k < p; k++) { io.print(let[0]); }
+			for (k = p; k < n; k++) { io.print(let[1]); }
+			io.println();
+		}
+		io.flush();
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-581D.mdx:188]: sae here

File: solutions/bronze/cf-581D.mdx [hunk 4]
```diff
@@ -89,4 +89,146 @@ int main() {
...
+}
+```
+
+</JavaSection>
+<PySection>
+
+```py
+a, b, c, d, e, f = list(map(int, input().split()))
+n, n2 = 1, a * b + c * d + e * f
+while n**2 < n2:
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-581D.mdx:201]:
```suggestion
while n * n < n2:
```

File: solutions/bronze/cf-581D.mdx [hunk 5]
```diff
@@ -89,4 +89,146 @@ int main() {
...
+	print(-1)
+	exit()
+l = sorted(
+	[
+		[max(a, b), min(a, b), "A"],
+		[max(c, d), min(c, d), "B"],
+		[max(e, f), min(e, f), "C"],
+	]
+)
+if l[2][0] != n:
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-581D.mdx:213]: space out your code please

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by shuklaji23** [comment]
@SansPapyrus683 Made all the requested changes. Please let me know if any other change is required.

**Review by SansPapyrus683 — APPROVED** [review]
crappy problem anyways, who cares

**Review by ryanchou-dev — COMMENTED** [review]
could you add comments? otherwise looks OK

**Comment by SansPapyrus683** [comment]
unfortunately i think we'll be taking #4569 over this, sorry about that!
feel free to submit prs for other problems tho :)

**Pull request closed without merging**